static/includes/create-service-console.md

```diff
@@ -0,0 +1,52 @@
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+
+1. Log in to [Aiven Console](https://console.aiven.io/) and open your project.
+
+1. From your project, on the left-side menu, click <ConsoleLabel name="services"/> > **Create service**.
+
+1. From the **Select service** page,
+   click {props.serviceType ? <strong>{props.serviceType}</strong> : 'the service type of your choice'}.
+
+1. Select the cloud provider and region to host your service on.
+
+   :::note
+   The pricing for the same service can vary between different
+   providers and regions. The service summary shows you the pricing
+   for your selected options.
+   :::
+
+1. Select a service plan.
+
+   :::note
+   This determines the number of servers and the memory,
+   CPU, and disk resources allocated to your service. See
+   [Plans & Pricing](https://aiven.io/pricing).
+   :::
+
+1. Optional: Add [disk storage](/docs/platform/howto/add-storage-space).
+
+1. Enter a name for your service.
```

Comment:
Is there a way to hide this step for Flink and Metrics, as you cannot fork these services?